Automattic/jetpack
# VideoPress: set Video URL in replace control based on privacy

This PR sets the video URL in the Replace control based on the video privacy, switching between `videopress.com` to `files.videopress.com`

Fixes #

## Proposed changes:

* VideoPress: set Video URL in the replac- control, based on privacy

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Go to the block editor
* Create a video block instance
* Set the video
* Confirm that, based on the video privacy, the Replace control shows the proper URL

Private | Public
------|------
<img width="636" alt="image" src="https://user-images.githubusercontent.com/77539/211579262-a2c5f7c1-c66e-4dcd-9232-ab9f2e0dfd5e.png"> | <img width="635" alt="image" src="https://user-images.githubusercontent.com/77539/211579670-49f1e191-5b3c-446d-9be3-33f681a393fe.png">
<img width="635" alt="image" src="https://user-images.githubusercontent.com/77539/211576546-569268e5-c5ce-46d7-a2f0-1152dcf7bd27.png"> | <img width="635" alt="image" src="https://user-images.githubusercontent.com/77539/211576677-c729d31e-8562-4480-afc6-d569135c15d6.png">
